src/test/java/com/datalogics/pdf/samples/forms/FillFormTest.java

```diff
@@ -31,6 +31,7 @@
  */
 public class FillFormTest extends SampleTest {
 
+    private static final String BAD_XML_INPUT = "xfa_bad_data.xml";
```

Comment:
Can you rename this to something more specific like maybe *missing_xfa_tags_.xml*? Just for maintenance purposes it would be good to know what is "bad" about it at a glance.